tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -181,7 +181,7 @@ install_image() {
 
 	local osbuilder_last_commit="$(get_last_modification "${repo_root_dir}/tools/osbuilder")"
 	local guest_image_last_commit="$(get_last_modification "${repo_root_dir}/tools/packaging/guest-image")"
-	local agent_last_commit="$(get_last_modification "${repo_root_dir}/src/agent")"
+	local agent_last_commit="$(get_last_modification "${repo_root_dir}/tools/packaging/static-build/agent")"
```

Comment:
@BbolroC - I think we need to monitor the static-build directory for changes to the builder image *and* the source directory incase that has changed, so that we trigger the rootfs re-build in either case? Sorry if I wasn't clear in my comment, I've been struggling to get it all aligned in my head

Reply:
Yes, you are right. The source itself as well as the static-build should be taken into account for the rootfs. Very precious comment. I will update the PR then. 😉